<launch>
	<param name="vel_max" type="double" value="0.8" />
	<param name="dist_stop" type="double" value="0.3" />
	<param name="dist_decelerate" type="double" value="2.10" />
	<node pkg="tutorial_kinematics" type="src/tutorial_kinematics_node.cpp" name="tutorial_kinematics_node" output="screen"/>
</launch>
